template/graphql/modules/scalars.ts

```diff
@@ -0,0 +1,5 @@
+import { DateTimeResolver, JSONObjectResolver } from 'graphql-scalars';
+import { asNexusMethod } from '@nexus/schema';
+
+export const jsonScalar = asNexusMethod(JSONObjectResolver, 'json');
+export const dateTimeScalar = asNexusMethod(DateTimeResolver, 'date');
```

Comment:
these scalars were present in framework. others are available should we need them